keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/ec2/resource/SecurityGroupHandler.kt

```diff
@@ -53,7 +56,7 @@ class SecurityGroupHandler(
   private val orcaService: OrcaService,
   override val objectMapper: ObjectMapper,
   override val normalizers: List<ResourceNormalizer<*>>
-) : ResourceHandler<SecurityGroup> {
+) : ResolvableResourceHandler<SecurityGroup, Map<String, SecurityGroup>> {
```

Comment:
Should we use different types here like the difference between `ClusterSpec` and `ServerGroup`? It seems confusing that the handler can return multiple single-region security groups, but their type makes it look like they could contain multiple regions.

Reply:
Went ahead and refactored this to split out `SecurityGroupSpec` (multi-region) from `SecurityGroup` (single-region). After doing it, I agree this is cleaner and makes sense going forward.

Reply:
Awesome